pkg/currency/constants.go

```diff
@@ -0,0 +1,3 @@
+package currency
+
+var kvStoreName = "currency"
```

Comment:
If you don't plan to change that name, you could make it a `const`.

Reply:
done